### PR TITLE
fix: icon sizes

### LINKED ISSUE #8997: [Bug] Smaller AnkiDroid icon, and magnifying glass icon after 2.15.0
Maybe it's two different bugs, but since it concerns icons that became smaller after the 2.15.0 update I put them together, as I suspect they might come from the same root cause.

Bug 1: **The AnkiDroid icon** (the tilted white star with round edges) **is smaller in the notification bar than before**, even though it takes the same amount of space width-wise, thus it would be better if it went back to its previous size, taking advantage of the full space it can have, thus making it more visible. It is currently harder at a quick glance, or for people with bad eyesight, to see what that icon is that indicates that you have cards to do.

Bug 2: **The magnifying glass icon** that appears at the top right of a card, that allows you to launch a search of the selected item (in an external app for instance), that you get after copying something on a card being reviewed, and then tapping anywhere on the screen, **is smaller than before**. It is harder to select than before, so it would be better if it went back to its previous size.

###### Reproduction Steps
Bug 1:
1. Have cards due
2. The AnkiDroid icon is in the notification bar 

Bug 2:
1. Have a card open for review
2. Select any item (e.g. a character, a word) by long pressing, and then on the pop up menu selecting 'Copy'
3. Tap anywhere on the card
4. The magnifying glass icon appears at the top right of the card. (If you press it launches a search for instance in another app of what you selected)

###### Expected Result
Bug 1:
The icon is the same size as before.
Bug 2:
The icon the same size as before.

###### Actual Result
Bug 1:
The icon smaller than before.
Bug 1:
The icon smaller than before.

###### Debug info
AnkiDroid Version = 2.15.2

Android Version = 10

Manufacturer = samsung

Model = SM-G960F

Hardware = samsungexynos9810

Webview User Agent = Mozilla/5.0 (Linux; Android 10; SM-G960F Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/90.0.4430.210 Mobile Safari/537.36

ACRA UUID = 64cf8984-4f06-4def-9281-10a05fddae3e

Scheduler = std

Crash Reports Enabled = true

DatabaseV2 Enabled = true

###### Research
- [x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by david-allison-1**
## Purpose / Description
In switching from SVG to PNG, some icons were resized incorrectly:

* Notification Icon
* Lookup icon

## Fixes
Fixes #8997

## Approach
* Convert the notification icon to svg, ensure it takes up all the viewport and re-export. Used https://shapeshifter.design/ for the conversion.
* The lookup icon moved from `wrap_content` to a reasonable size on my phone,
My phone had the pngs at 160px (~58dp), so round them up to 60 and call it a day

## How Has This Been Tested?
Android 11 (Pixel 3A)

Before/After from 2.14.6 and 2.15

image
image
image

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/drawable/ic_stat_notify.xml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
....1 C 35.5 36.5 34.6 39 34.1 39.7 C 33.6 40.4 31.4 42 28.3 43.3 C 25.2 44.7 21.3 46 17.4 47.4 C 13.5 48.8 9.7 50.3 6.4 52.3 C 3.2 54.3 0 57.5 0 62 C 0 66.5 3.1 69.6 6.4 71.7 C 9.7 73.8 13.4 75.2 17.3 76.7 C 21.2 78.1 25.1 79.4 28.2 80.8 C 31.3 82.2 33.5 83.7 34 84.4 C 34.5 85.1 35.3 87.7 35.6 91.1 C 35.9 94.5 36 98.6 36.1 102.8 C 36.2 106.9 36.5 111 37.4 114.7 C 38.3 118.4 40.4 122.4 44.6 123.8 C 48.8 125.2 52.8 123.2 55.8 120.7 C 58.8 118.2 61.4 115.1 63.9 111.9 C 66.4 108.7 68.9 105.3 71.2 102.8 C 73.5 100.3 75.6 98.6 76.5 98.4 C 77.3 98.1 80 98.2 83.3 98.9 C 86.6 99.6 90.6 100.9 94.5 102 C 98.5 103.1 102.4 104.2 106.2 104.4 C 110 104.6 114.4 103.9 117.1 100.3 C 119.7 96.7 119.1 92.3 117.7 88.7 C 116.3 85.1 114.1 81.7 111.8 78.3 C 109.5 74.9 107.1 71.5 105.4 68.6 C 103.7 65.7 102.8 63.1 102.8 62.2 C 102.8 61.2 103.9 58.2 106.1 54.8 C 108.2 51.4 111.1 47.5 113.6 43.7 C 115.6 40.7 117.3 37.6 118.4 34.5 C 119.4 31.3 119.7 27.4 117.4 24.2 C 114.8 20.6 110.4 19.9 106.5 20.1 C 102.7 20.3 98.7 21.4 94.7 22.5 C 90.7 23.6 86.8 24.9 83.5 25.6 C 80.2 26.3 77.5 26.3 76.7 26.1 C 75.9 25.9 73.7 24.2 71.4 21.7 C 69.1 19.1 66.7 15.8 64.2 12.5 C 61.7 9.2 59.1 6.1 56.1 3.6 C 53.8 1.7 51 0.1 48 0 Z"
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/drawable/ic_stat_notify.xml:10]: Note: this produces an IDE warning about long path data

**Review by mikehardy — APPROVED** [review]
Going to trust the screenshots and your work here - certainly doesn't appear to be a regression. Every once in a while you re-look at your own work and change something so I'll leave it for you to merge @david-allison-1 but otherwise I would merge right now so feel free. I'll pick to 2.15 after - we've got a few things landing and as long as they're on the milestone and not tagged as picked I won't miss them before release so no worries there re: communication on merge + pick etc

**Comment by david-allison-1** [comment]
I don't use either of these features, so won't have additional feedback. Let's go 🚀

**Comment by viivgit** [comment]
I can confirmed, both icons are back to their normal sizes! thanks for the fix!

**Pull request merged**